mne/source_estimate.py

```diff
@@ -447,13 +447,21 @@ class _BaseSourceEstimate(ToDataFrameMixin, TimeMixin):
 
     Parameters
     ----------
-    data : array, shape (n_dipoles, n_times) | tuple, shape (2,)
-        The data in source space. The data can either be a single array or
-        a tuple with two arrays: "kernel" shape (n_vertices, n_sensors) and
-        "sens_data" shape (n_sensors, n_times). In this case, the source
-        space data corresponds to "numpy.dot(kernel, sens_data)".
-    vertices : array | list of array
-        Vertex numbers corresponding to the data.
+    data : array of shape (n_dipoles, n_times) | tuple, shape (2,)
+        The data in source space. When it is a single array, the
+        left hemisphere is stored in data[:vertices[0]] and the right
```

Comment:
this is not correct. Eventually it should be `data[:len(vertices[0])]`

Reply:
good catch!